### ISSUE TITLE
notification when online.

### ISSUE THREAD

**Issue by Heinrich-XIAO** [Heinrich-XIAO/Tweetor#512]
I think we shouod get a notification when someone gets online.

**Comment by oblivikun** [comment]
No. this is a horrible idea because you will be spammed with notifications. It might be a good idea to add a "watchlist" though so it will send a notification if a specified user is online
